operations/enable-nfs-ldap.yml

```diff
@@ -20,6 +20,9 @@
 - type: replace
   path: /instance_groups/name=diego-cell/jobs/name=nfsv3driver/properties/nfsv3driver/allowed-in-source?
   value: ""
+- type: replace
+  path: /instance_groups/name=diego-cell/jobs/name=nfsv3driver/properties/nfsv3driver/ldap_ca_cert?
+  value: ((ldap_test_server_ssl.ca))
```

Comment:
The certificate `ldap_test_server_ssl` is defined in [enable-nfs-test-ldapserver.yml](./operations/test/enable-nfs-test-ldapserver.yml), but that ops file is only for test purposes. Your intention is to make the ldap's SSL CA certificate configurable, right? Should we use another name for the CA (without "test")? You should also enhance the [vars-enable-nfs-ldap.yml](./operations/example-vars-files/vars-enable-nfs-ldap.yml) file to include the CA. See e.g. [vars-use-trusted-ca-cert-for-apps.yml](./operations/example-vars-files/vars-use-trusted-ca-cert-for-apps.yml).

Reply:
@jochenehret I have added an example value for the CA and updated the PR.

I don't have a strong opinions about the naming convention, since you can't use `enable-nfs-ldap.yml` without `enabled-nfs-test-ldapserver.yml`. Even if your ldap server is external to the foundation, you'd have to have it deployed with a cert.

Reply:
You should be able to use `enable-nfs-ldap.yml` without `enable-nfs-test-ldapserver.yml`. The latter ops file deploys a test ldap server and also generates a test SSL certificate. For a productive setup, you just need `enable-nfs-ldap.yml` and provide a SSL certificate. That certificate would have to be generated externally (not BOSH managed).